accord-core/src/main/java/accord/impl/AbstractConfigurationService.java

```diff
@@ -168,15 +196,25 @@ Topology topologyFor(long epoch)
             return getOrCreate(epoch).topology;
         }
 
+        Topology topologyForLastReceived()
```

Comment:
I did this as I was planning to just do `synchronized`, but now that `lastReceived` can be accessed without locks it wasn't needed... kept it as I still like it but can revert to `epochs.topologyFor(epochs. lastReceived())` if desired